components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/components/Application/Create/ApplicationCreate.jsx

```diff
@@ -218,7 +246,26 @@ class ApplicationCreate extends Component {
                         <Row>
                             <Col>
                                 <div className="stepper-header">
-
+                                    <Row>
+                                        <Col>
+                                            <div className="stepper-header-content">
+                                                <div className="step-index">1</div>
+                                                <div className="step-header"> General Info</div>
+                                            </div>
+                                        </Col>
+                                        <Col>
+                                            <span className="step-index">2</span>
+                                            <span className="step-header"> Select Platform</span>
```

Comment:
Please use localization based on react-intl as megala has already integrated it.